terraform/module-ses/outputs.tf

```diff
@@ -21,3 +21,7 @@ output "iam_ses_smtp_user_secret" {
 output "aws_sqs_queue_url" {
   value = "${element(concat(aws_sqs_queue.email_delivery_queue.*.id, list("")), 0)}"
 }
+
+output "elasticache_fqdn" {
+  value = ["${aws_route53_record.redis.fqdn}"]
```

Comment:
I m not sure if we should put `[ ]`.  If a user follow the doc, he will not expect an array. Maybe use a different workaround for output optional